vignettes/quickstart.Rmd

```diff
@@ -332,90 +347,97 @@ For some applications we have prior knowledge of sets of words that are indicati
 For example, let's look at how words associated with terrorism and words associated with the economy vary by President in the inaugural speeches corpus. From the original corpus, we select Presidents since Clinton:
 
 ```{r}
-recent_corpus <- corpus_subset(data_corpus_inaugural, Year > 1991)
+corp_inaug_post1991 <- corpus_subset(data_corpus_inaugural, Year > 1991)
 ```
 
 Now we define a demonstration dictionary:
 ```{r}
-my_dict <- dictionary(list(terror = c("terrorism", "terrorists", "threat"),
-                          economy = c("jobs", "business", "grow", "work")))
+dict <- dictionary(list(terror = c("terrorism", "terrorists", "threat"),
+                        economy = c("jobs", "business", "grow", "work")))
 ```
 
 
 We can use the dictionary when making the dfm:
 ```{r}
-by_pres_mat <- dfm(recent_corpus, dictionary = my_dict)
-by_pres_mat
+dfmat_inaug_post1991_dict <- dfm(corp_inaug_post1991, dictionary = dict)
+dfmat_inaug_post1991_dict
 ```
 
 The constructor function `dictionary()` also works with two common "foreign" dictionary formats: the 
-LIWC and Provalis Research's Wordstat format.  For instance, we can load the LIWC and apply this to the Presidential inaugural speech corpus:
+LIWC and Provalis Research's Wordstat format. For instance, we can load the LIWC and apply this to the Presidential inaugural speech corpus:
 ```{r, eval = FALSE}
-liwcdict <- dictionary(file = "~/Dropbox/QUANTESS/dictionaries/LIWC/LIWC2001_English.dic",
+dictliwc <- dictionary(file = "~/Dropbox/QUANTESS/dictionaries/LIWC/LIWC2001_English.dic",
                        format = "LIWC")
-liwcdfm <- dfm(data_corpus_inaugural[52:58], dictionary = liwcdict)
-liwcdfm[, 1:10]
+dfmat_inaug_subset <- dfm(data_corpus_inaugural[52:58], dictionary = dictliwc)
+dfmat_inaug_subset[, 1:10]
 ```
 
-# Further Examples
+# Further examples
 
 ## Similarities between texts
 
 ```{r fig.width = 6}
-pres_dfm <- dfm(corpus_subset(data_corpus_inaugural, Year > 1980), 
-               remove = stopwords("english"), stem = TRUE, remove_punct = TRUE)
-obama_simil <- textstat_simil(pres_dfm, c("2009-Obama" , "2013-Obama"), 
-                             margin = "documents", method = "cosine")
-obama_simil
-# dotchart(as.list(obamaSimil)$"2009-Obama", xlab = "Cosine similarity")
+dfmat_inaug_post1980 <- dfm(corpus_subset(data_corpus_inaugural, Year > 1980),
+                            remove = stopwords("english"), stem = TRUE, remove_punct = TRUE)
+tstat_obama <- textstat_simil(dfmat_inaug_post1980, 
+                              c("2009-Obama" , "2013-Obama"), 
+                              margin = "documents", method = "cosine")
+tstat_obama
+# dotchart(as.list(tstat_obama)$"2009-Obama", xlab = "Cosine similarity")
 ```
 
 We can use these distances to plot a dendrogram, clustering presidents:
-```{r, fig.width = 10, fig.height = 7, eval = FALSE}
```

Comment:
The problem here is that because **quanteda.corpora** is not published on CRAN, we cannot have the vignette use it, unless we park the sotu corpus binary somewhere and load it using a `load(url(...))` call.

Reply:
What about just using `data_corpus_inaugural` instead? We only use `data_corpus_sotu` for the dendrogram and and the **topicmodels** example. I could change the code accordingly if you agree, @kbenoit.

Reply:
Or, we load a binary that we park on the Quanteda.org website. The sotu speeches cluster very nicely, better than the inaugural speeches.

Reply:
Ok, let's store the SOTU corpus on the quanteda.org website at (https://quanteda.org/data/...) and use `load(url(...))`. @koheiw, could you take care of this?

Reply:
I fixed this using the following:
```r
data_corpus_sotu <- readRDS(url("https://quanteda.org/data/data_corpus_sotu.rds"))
```